sdk/core/System.ClientModel/tests/Pipeline/ClientPipelineTests.cs

```diff
@@ -323,4 +323,14 @@ public async Task RequestOptionsCanCustomizePipeline()
         Assert.AreEqual("Response:RetryPolicy", observations[index++]);
         Assert.AreEqual("Response:A", observations[index++]);
     }
+
+    [Test]
+    public void CanCreateWithLoggingPolicy()
+    {
+        ClientPipelineOptions options = new ClientPipelineOptions();
+        options.LoggingPolicy = new ClientLoggingPolicy();
+        ClientPipeline pipeline = ClientPipeline.Create(options);
+
+        // TODO: check sanitizer (internal) is set
```

Comment:
It would be good to add functional tests either in the pipeline tests (here) or in the policy tests.  How can this feature be observed through public APIs?